vita3k/gui/src/settings_dialog.cpp

```diff
@@ -77,6 +76,10 @@ void get_modules_list(GuiState &gui, HostState &host)
                 gui.modules_list.push_back(lle.filename().replace_extension().string());
         }
     }
+
+    gui.modules_selected.resize(gui.modules_list.size());
+    for (size_t i = 0; i < gui.modules_list.size(); ++i)
+        gui.modules_selected[i] = std::find(host.cfg.lle_modules.begin(), host.cfg.lle_modules.end(), gui.modules_list[i]) != host.cfg.lle_modules.end();
```

Comment:
secondary compared my message, for maybe run iut only if module folder is not empty/no exist ^^
```suggestion
        }
            gui.modules_selected.resize(gui.modules_list.size());
    for (size_t i = 0; i < gui.modules_list.size(); ++i)
        gui.modules_selected[i] = std::find(host.cfg.lle_modules.begin(), host.cfg.lle_modules.end(), gui.modules_list[i]) != host.cfg.lle_modules.end();
    }
```